bridge/elrond/client.go

```diff
@@ -132,14 +156,14 @@ func (c *Client) GetPending(context.Context) *bridge.Batch {
 	var transactions []*bridge.DepositTransaction
 	for i := 1; i < len(responseData); i += 6 {
 		amount := new(big.Int).SetBytes(responseData[i+5])
-		blockNonce, err := strconv.ParseInt(hex.EncodeToString(responseData[i]), 16, 64)
-		if err != nil {
-			c.log.Error(err.Error())
+		blockNonce, errParse := strconv.ParseInt(hex.EncodeToString(responseData[i]), 16, 64)
```

Comment:
what are `i += 6` and `i+5` ? ..

Reply:
I do not know. Will try to add some checks so the code won't panic

Reply:
added consts & checks